WMF Framework/Configuration.swift

```diff
@@ -41,7 +42,7 @@ public class Configuration: NSObject {
         #if WMF_LOCAL
         return Configuration.local(options: [.localPCS, .localAnnouncements])
         #elseif WMF_STAGING
-        return Configuration.staging(options: [.appsLabsforPCS])
+        return Configuration.staging(options: [.betaCluster])
```

Comment:
To do `appsLabsForPCS`, we'd adjust the value here, right? Is it worth leaving a very explicit note or a commented out line of code or something like that?

Reply:
Pushed a commit with a descriptive comment, let me know if you'd like me to tweak it further. Thanks!